editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper.tsx

```diff
@@ -33,6 +33,11 @@ export function buildSpyWrappedElement(
   }
   const childrenElementsOrNull = childrenElements.length > 0 ? childrenElements : null
   const spyCallback = (reportedProps: any) => {
+    /** This is not so nice, but the way to know if something is an emotion component is
+     * that it adds some extra properties to the Element itself, like __emotion_base,
+     * TODO move this out of metadata once we have syledcomponent editing
+     */
+    const isEmotionComponent = Element['__emotion_base'] != null
```

Comment:
Does emotion export a const for this key (or any other key we can rely on)? If not, please can you add a test to ensure we're correctly recording `isEmotionComponent: true` for a styled component so that we can catch if they change this key.